preprocessors/Emilia/models/separate_fast.py

```diff
@@ -0,0 +1,288 @@
+# Copyright (c) 2024 Amphion.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
```

Comment:
@yuantuo666  same as my comments above.

Reply:
https://github.com/seanghay/uvr-mdx-infer The source code did not claim the license. I temporaily added a MIT license here since the https://github.com/Anjok07/ultimatevocalremovergui repository claimed to be MIT-license.

Reply:
We might need to requests a issue to original repository to ask about the license and relevent right about this.

Reply:
we can claim that ``this code is modified from xxxx.py`` like the following example: https://github.com/open-mmlab/Amphion/blob/5dfe9fd663a96f69854bb060fa52fe15946fe690/models/tts/valle/valle.py#L6